tools/android/packaging/xbmc/src/XBMCJsonRPC.java.in

```diff
@@ -24,15 +24,17 @@ import android.net.Uri;
 import android.provider.BaseColumns;
 import android.util.Log;
 
+import @APP_PACKAGE@.XBMCTextureCache;
+import @APP_PACKAGE@.content.XBMCFileContentProvider;
+import @APP_PACKAGE@.content.XBMCYTDLContentProvider;
```

Comment:
Unused import statement (XBMCYTDLContentProvider)

Reply:
`import @APP_PACKAGE@.content.XBMCFileContentProvider;` must be

Reply:
Ah sorry, should have read your comment properly :slightly_smiling_face: